tilequeue/command.py

```diff
@@ -1074,6 +1074,19 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
                     coord_marshall_int(deserialize_coord(l.strip()))
                     for l in f
                 )
+        elif 's3' in info:
+            from boto import connect_s3
+            from boto.s3.bucket import Bucket
+            s3_conn = connect_s3()
+            bucket = Bucket(s3_conn, info['s3']['bucket'])
+            key = bucket.get_key(info['s3']['key'])
+            raw_coord_data = key.get_contents_as_string()
```

Comment:
Good idea, but when I tried it locally, it seemed to iterate over "chunks" rather than lines. Is there a way to tell it to iterate over lines? Or is it better to just stick with the way it's done currently? I'm confident that this list will fit into memory :)